### PR TITLE
Add references support to download data script

### PR DESCRIPTION
**Pull request by dcolligan**
Issue #578

### CODE REVIEW

File: scripts/download_data.py [hunk 1]
```diff
@@ -231,6 +258,31 @@ def downloadBams(self):
...
+        cleanDir()
+        baseUrl = 'http://www.ebi.ac.uk/ena/data/view/'
+        for chromosome in self.chromosomes:
+            accession = self.accessions[chromosome]
+            path = os.path.join(baseUrl, accession)
+            minPos = self.chromMinMax.getMinPos(chromosome)
+            maxPos = self.chromMinMax.getMaxPos(chromosome)
+            args = urllib.urlencode({
+                'display': 'fasta',
+                'range': '{}-{}'.format(minPos, maxPos)})
```
Comment (jeromekelleher) [inline scripts/download_data.py]: I think this should be ``(0, maxPos)``; otherwise, we'll hit some nasty offsetting problems.

**Comment by jeromekelleher** [comment]
This is very nice, thanks @dcolligan. Other than the minor point above, it's all good.

**Comment by dcolligan** [comment]
@jeromekelleher updated

**Comment by jeromekelleher** [comment]
Thanks @dcolligan, +1.

**Pull request merged**